### PR TITLE
Refinace PH table as step

### PR DESCRIPTION
**Pull request by piekczyk**
# Refinance PH as step

<please insert a shortcut link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>

- added PH step component which we will use in one of the refinance steps
  
## How to test 🧪
  <Please explain how to test your changes>

- nothing to test now

### CODE REVIEW

File: features/refinance/RefinanceContext.tsx [hunk 1]
```diff
@@ -72,17 +118,61 @@ export function RefinanceContextProvider({
...
+        collateralTokenAmount,
+        debtTokenAmount,
+        liquidationPrice,
+        positionId,
+        ltv,
+      },
+      poolData: {
+        poolId,
+        borrowRate,
+        collateralTokenSymbol,
```
Comment (piotrwitek) [inline features/refinance/RefinanceContext.tsx:170]: this is already in collateralTokenAmount

File: features/refinance/RefinanceContext.tsx [hunk 2]
```diff
@@ -72,17 +118,61 @@ export function RefinanceContextProvider({
...
+        debtTokenSymbol,
```
Comment (piotrwitek) [inline features/refinance/RefinanceContext.tsx:171]: this is already in debtTokenAmount

File: features/refinance/RefinanceContext.tsx [hunk 3]
```diff
@@ -1,37 +1,90 @@
...
+  steps: RefinanceSidebarStep[]
+  txStatus?: TxStatus
+  setIsFlowStateReady: Dispatch<SetStateAction<boolean>>
+  setStep: (step: RefinanceSidebarStep) => void
+  setNextStep: () => void
+  setPrevStep: () => void
+}
+
 export type RefinanceContextInput = {
-  positionId: PositionId
```
Comment (piotrwitek) [inline features/refinance/RefinanceContext.tsx:9]: do we need that nesting? not sure what is the benefit... for me it's harder to access props as there is extra nesting to remember about

**Review by piotrwitek — COMMENTED** [review]
Added few comments

File: features/refinance/RefinanceContext.tsx [hunk 4]
```diff
@@ -1,37 +1,90 @@
...
-  collateralTokenSymbol: string
-  debtTokenSymbol: string
-  collateralAmount: string
-  debtAmount: string
-  chainId: number
-  slippage: number
-  tokenPrices: Record<string, string>
-  address?: string
-  liquidationPrice: string
+  poolData: {
```
Comment (piotrwitek) [inline features/refinance/RefinanceContext.tsx:28]: do we need that nesting? not sure what is the benefit of this... for me it's harder to access props as there is extra nesting to remember about

File: features/refinance/RefinanceContext.tsx (hunk 2 above)
Comment (piekczyk) [reply features/refinance/RefinanceContext.tsx:171]: it feels unintuitive to not have these symbols on pool data

**Review by piotrwitek — APPROVED** [review]
looks good

**Pull request merged**